api/internal/managers/installation/config_test.go

```diff
@@ -356,8 +356,6 @@ func TestConfigureHost(t *testing.T) {
 					hum.On("ConfigureHost", mock.Anything, hostutils.InitForInstallOptions{
 						LicenseFile:  "license.yaml",
 						AirgapBundle: "bundle.tar",
-						PodCIDR:      "10.0.0.0/16",
-						ServiceCIDR:  "10.1.0.0/16",
 					}).Return(nil),
```

Comment:
Should we validate the RC param VS keeping the `mock.Anything`?

Reply:
done